templates/workspaces/base/porter.yaml

```diff
@@ -235,6 +240,7 @@ upgrade:
         arm_environment: ${ bundle.parameters.arm_environment }
         enable_cmk_encryption: ${ bundle.parameters.enable_cmk_encryption }
         key_store_id: ${ bundle.parameters.key_store_id }
+        storage_account_redundancy: ${ bundle.parameters.storage_account_redundancy }
```

Comment:
It might not be important that much but for consistency please also pass this parameter to uninstall too